### PR TITLE
Update master with changes made to Syriaca.org for use with eXist version 5

### PR DESCRIPTION
**Pull request by wsalesky**
@nathangibson Please let me know if it is okay to merge this.

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project default="xar" name="usaybia">
+<project default="xar" name="srophe">
     <xmlproperty file="expath-pkg.xml"/>
     <property name="project.version" value="${package(version)}"/>
-    <property name="project.app" value="usaybia"/>
+    <property name="project.app" value="srophe"/>
```
Comment (nathangibson) [inline build.xml]: Does this need to be kept as "usaybia" or "usaybia.net" or will it function okay like this?

**Comment by nathangibson** [comment]
@wsalesky Good! 
But I'm confused are these changes going into the srophe master branch?? Or into the usaybia fork?

**Comment by nathangibson** [comment]
Assuming it's being merged into the correct branch you can go ahead and merge.

**Comment by wsalesky** [comment]
@nathangibson You are correct, I am merging into the wrong base. Thanks for catching that!

**Pull request closed without merging**